tools/generate-code.py

```diff
@@ -51,16 +53,16 @@ def generate_towire_field(field, allfields, lang):
                           lenfield=field.fieldtype.len_for[0].name), file=ofile)
     else:
         if lang == 'js':
-            print('    buf = Buffer.concat([buf, towire_{ftype}(value[_n])]);'
+            print('    buf = Buffer.concat([buf, towire_{ftype}(value[_n++])]);'
                   .format(ftype=field.fieldtype.name), file=ofile)
         elif lang == 'py':
             print('    buf += towire_{ftype}(value["{fname}"])'
                   .format(fname=field.name,
                           ftype=field.fieldtype.name), file=ofile)
-    if lang == 'js':
-        print('    _n++;', file=ofile)
-    elif lang == 'py':
-        print('    _n += 1', file=ofile)
+    # if lang == 'js':
+    #     print('    _n++;', file=ofile)
+    # elif lang == 'py':
+    #     print('    _n += 1', file=ofile)
```

Comment:
I generally dislike leaving commented code around; if it passes review, it should be sufficient that the old version is in git if we need it.